site/pt/blog/new-in-devtools-102/index.md

```diff
@@ -28,126 +27,156 @@ draft: true
 
 
 <!-- ## Console updates  {: #console } -->
+## Atualizações no Console {: #console }
 
 <!-- ### Handle script execution errors in the Console {: #errors } -->
+### Tratamento de erros de execução de script no Console {: #errors }
 
 <!-- Errors during script evaluation in the Console now generate proper error events that trigger the `window.onerror` handler and are dispatched as `"error"` events on the window object. -->
+Erros durante a execução de um script no Console agora geram os eventos corretos de erro que ativam a chamada `window.onerror` e são enviados como eventos do tipo `"error"` para o objeto `window`.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/gBtY4zD39SPizfcCGJJW.png", alt="Handle script execution errors in the Console", width="800", height="487" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/gBtY4zD39SPizfcCGJJW.png", alt="Tratamento de erros de execução de script no Console", width="800", height="487" %}
 
 {# https://chromium.googlesource.com/v8/v8/+/56cfdd68c731c53d016326b890b56b5c30098998 #}
 
-Chromium issue: [1295750](https://crbug.com/1295750)
+Issue relacionada: [1295750](https://crbug.com/1295750)
 
 
 <!-- ### Commit live expression with Enter {: #live-expression } -->
-
+### Enter para enviar uma *live expression*  {: #live-expression }
 <!-- Once you finish typing a [live expression](/blog/new-in-devtools-70/#watch), you can click `Enter` to commit it. Previously, hitting Enter resulted in adding new lines. This is inconsistent with other parts of the DevTools.  -->
-
+Quando você terminar de digitar uma [live expression](/blog/new-in-devtools-70/#watch), você pode apertar `Enter` para enviá-la. Anteriormente, apertar `Enter` resultava na adição de novas linhas. Isso é inconsistente com outras partes do DevTools.
 <!-- To add a new line in the **live expression** editor, use `Shift` + `Enter` instead. -->
+Para adicionar uma nova linha no editor de **live expressions** use `Shift` + `Enter`.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yB7m2052mYzgsRgjIMvs.png", alt="Commit live expression with Enter", width="800", height="541" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/yB7m2052mYzgsRgjIMvs.png", alt="Enter para enviar uma live expression", width="800", height="541" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/f6f73b8d9eedbc5b6006e61c3be0d843188eac55 #}
 
-Chromium issue: [1260744](https://crbug.com/1260744)
+Issue relacionada: [1260744](https://crbug.com/1260744)
 
 
 <!-- ## Cancel user flow recording at the start {: #recorder } -->
-
+## Cancelamento de gravações de fluxo de usuário no início {: #recorder }
 <!-- You can cancel the recording during the start of user flow recording. Previously, there was no option to cancel the recording. -->
+Você pode cancelar a gravação durante o início de uma gravação de fluxo de usuário. Anteriormente não havia essa opção.
```

Comment:
Você pode cancelar a gravação durante o início do registro do fluxo de usuário. Anteriormente não havia essa opção.

Reply:
```suggestion
Você pode cancelar a gravação durante o início de um registro de fluxo de usuário. Anteriormente não havia essa opção.
```